src/wp-includes/default-filters.php

```diff
@@ -757,4 +757,7 @@
 add_filter( 'rest_pre_insert_wp_template', 'inject_ignored_hooked_blocks_metadata_attributes' );
 add_filter( 'rest_pre_insert_wp_template_part', 'inject_ignored_hooked_blocks_metadata_attributes' );
 
+// Update ignoredHookedBlocks postmeta for wp_navigation post type.
+add_filter( 'rest_pre_insert_wp_navigation', 'update_ignored_hooked_blocks_postmeta' );
```

Comment:
I will need to add a `has_filter()` check for this in Gutenberg

Reply:
I've heard that the Gutenberg version that will be used to sync code into WP 6.6 will be an RC version that's going to be published next Friday (31st). I think that should be enough time to get the GB PR ready and merged, right? It's basically just that `has_filter` check IIUC 😊

Reply:
https://github.com/WordPress/gutenberg/pull/61903